### ISSUE TITLE
The entry in Perl is not the same as the entry in TRS-80 Basic

### ISSUE THREAD

**Issue by auraes** [pdxiv/PerlScott#6]
In Basic: `INPUT"TELL ME WHAT TO DO":TP$:PRINT`
There's a question mark (?) after the Input and no newlline before the player's entry.

> TELL ME WHAT TO DO? GET LAMP

In Perl:

> Tell me what to do
> GET LAMP

**Comment by pdxiv** [comment]
The question mark is default on the INPUT statement in TRS-80 Basic, i guess. If so - it should be added to the Perl version.

Another difference, is that the original Basic code was upper case only. The original game data files used upper-case only for text as well, so that was consistent back in 1978. However, newer game data files that the Perl version of the interpreter is compatible with use both upper and lower case, so i think i'll keep the current upper/lower case text in the Perl interpreter.

**Comment by auraes** [comment]
Yes, the question mark is by default on the INPUT instruction in TRS-80 Basic; I had tested it with the Basic of the TRS-80 emulator.

**Comment by pdxiv** [comment]
Great, thanks! I'll fix this.

Which emulator are you using for TRS-80? I still haven't found one that works very well for me, under Linux.

**Comment by auraes** [comment]
TRS-80 Model 1/3 Emulator for MS-DOS (under DosBox linux). It works well.
http://www.trs-80emulators.com/

**Comment by auraes** [comment]
In my first comment, I reproduced the error of the paper listing. There is a semicolon (;) after INPUT and not a colon (:)
In Basic: INPUT"TELL ME WHAT TO DO";TP$:PRINT

**Comment by heasm66** [comment]
Change the prompt to "Tell me what to do? " with a question mark and no CR (TRS-80 look & feel. See www.filfre.net/2011/06/adventureland-part-1/
